neo/Network/P2P/Payloads/StateRootsPayload.cs

```diff
@@ -0,0 +1,39 @@
+using Neo.IO;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace Neo.Network.P2P.Payloads
+{
+    public class StateRootsPayload : ISerializable
```

Comment:
These messages are not included in the enum https://github.com/neo-project/neo/blob/master/src/neo/Network/P2P/MessageCommand.cs ?

Reply:
The enum is defined in neo3 master. We don't have this in neo2.x.